frontend/console/src/features/modules/decls/DeclLink.tsx

```diff
@@ -3,12 +3,13 @@ import { useNavigate } from 'react-router-dom'
 import { useSchema } from '../../../api/schema/use-schema'
 import type { PullSchemaResponse } from '../../../protos/xyz/block/ftl/v1/ftl_pb.ts'
 import type { Decl } from '../../../protos/xyz/block/ftl/v1/schema/schema_pb'
+import { classNames } from '../../../utils'
 import { DeclSnippet } from './DeclSnippet'
 
 const SnippetContainer = ({ decl }: { decl: Decl }) => {
```

Comment:
Should we use a headless ui popup for this? It might help with ensuring this is always visible even if it's at the bottom of the page (or far right of the page) so it doesn't get cut off.

![Screenshot 2024-09-18 at 7 29 39 PM](https://github.com/user-attachments/assets/17316bb8-7a40-4077-87f6-09476b9a4640)

Reply:
This might become more of an issue when we're update the modules page to show more than just the schema. For example, we'll probably want to show traces/events etc. on the modules page as well, which would mean this might only be show on left or right half of the page. So having a popup that can adapt will be very useful.